tests/dummy/config/environment.js

```diff
@@ -19,6 +19,10 @@ module.exports = function(environment) {
       }
     },
 
+    // 'ember-yeti-table': {
+    //   pagination: true
+    // },
+
```

Comment:
This can be removed.

Reply:
This was used for texting. Left it in for easy re-use, figured it wouldn't hurt in the dummy app.